### ISSUE TITLE
Term description is missing from exported template

### ISSUE THREAD

**Issue by metagic** [SharePoint/PnP-Sites-Core#1631]
#### Category
[x] Bug

#### Environment
[x] SharePoint 2013

#### Expected or Desired Behavior
The <pnp:Term> Element produced from Get-PnPProvisioningTemplate or from ExportPnPTermGroupToXml should include the "Description" attribute.

**Comment by erwinvanhunen** [comment]
Hi, 

I just double checked by using both cmdlets and having set a description on a term on an SP2013 server. The description is included as expected. Please double check that you're using the latest version of the cmdlets: 2.23.1802.0.

**Comment by metagic** [comment]
Hi Erwin,
Thank you for looking into the issue. 2.23.1802.0 is exactly the cmdlets version I am using. Let me add the exact steps that produce the issue:
```
Import-PnPTermGroupFromXml -Path TestDescriptionImport.xml
$ctx = Get-PnPContext
$t = Get-PnPTerm -TermGroup "TestGroup" -TermSet "TestSet" -Identity "TestTerm"
$ctx.Load($t)
$ctx.ExecuteQuery()
$t.Description
Export-PnPTermGroupToXml -Identity "TestGroup" -Out TestDescriptionExport.xml
```
The input and output XMLs are attached: [TestDescriptionExport.zip](https://github.com/SharePoint/PnP-Sites-Core/files/1771419/TestDescriptionExport.zip)

Regards,
Nicolas

**Comment by metagic** [comment]
OK, is see this has been fixed in the meantime: #1561 

Thanks folks!
